pkg/cvo/availableupdates.go

```diff
@@ -25,16 +25,25 @@ import (
 
 const noArchitecture string = "NoArchitecture"
 const noChannel string = "NoChannel"
+const defaultUpstream string = "https://api.openshift.com/api/upgrades_info/v1/graph"
```

Comment:
Possibly a good opportunity to give this a better name in the code too. OSUS is the source of update data, I'm not convinced "upstream" expresses that well.

I would not change user-facing strings (like `spec.upstream` and `NoUpstream` condition type) though.

Reply:
I've pushed 64fb2264f9fefdfb81f02e9c3049d6299eae0e77 -> 04c32b523a21d7f1025b877de51bb8b09c4376ce with a pivot to `--update-service`, `updateService`, etc.